packages/docs/src/components/ResponsiveExample.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState, useRef, useEffect } from 'react';
+import classnames from 'classnames';
+
+const breakpointOpts = {
```

Comment:
FYI I have these in tokens as :

```
  'width-xs': '0px',
  'width-sm': '544px',
  'width-md': '768px',
  'width-lg': '1024px',
  'width-xl': '1280px',
```

Which is inherited [from our css](https://github.com/CMSgov/design-system/blob/7e1c2bf193fcd62d461071073b69cc6d05c2d757/packages/design-system/src/styles/settings/variables/_layout.scss#L38), I wonder why `width-xs` is zero..